src/components/datePicker/index.module.css

```diff
@@ -12,12 +12,6 @@
     @apply shadow-hard rounded-6 bg-white leading-xs w-12/12 p-10;
 }
 
-@screen xxs {
-    :local .react-calendar {
-        min-width: 346px;
```

Comment:
does anyone remember why we need this value?

Reply:
to ensures the files stay square and don't become rectangular, simply dropping it will not make that sure any more, if it does not has the required width we probably need to lower font and tile size one breakpoint before...
(Only gave it a quick test there might be more UI related issues when just dropping it)

Before:
![Screenshot 2021-05-05 at 17 55 19](https://user-images.githubusercontent.com/3371760/117171721-6b97aa00-adcb-11eb-93ba-58062f9e419d.png)

After
![Screenshot 2021-05-05 at 17 55 41](https://user-images.githubusercontent.com/3371760/117171719-69cde680-adcb-11eb-8d50-5a02dd14ce4d.png)

After:  (font not centered)
![Screenshot 2021-05-05 at 17 48 18](https://user-images.githubusercontent.com/3371760/117171725-6c304080-adcb-11eb-970a-2a5c853e23b8.png)

Reply:
Something that just came to my mind, using the calendar as a dropdown option is only one use case for the component. As a component, we might wanna use it for other purposes where the calendar is not spaced. In that way, the component does not push itself to the right, it's the listings project that does that. I am not fully sure about the desired behaviour from a UX perspective in the concrete use case but the component will never become wider than the screen. In that way, if the component ensures its UI works as intended with the min-width and would look perfectly fine when given the available space before becoming smaller...

![image](https://user-images.githubusercontent.com/3371760/117175421-1f4e6900-adcf-11eb-950b-8c78dc1dfef6.png)

Reply:
thanks for checking, the issue is that the calendar can be wider than the input element, and might look weird, I will ask the designer if its fine or not, if its not a bug then I will just close this PR

Reply:
Yes, I saw the ticket. (Not sure if the intended solution is that the calendar should open centre to the input in that case or we should show the smaller version of the calendar) What I tried to explain is that we can not just drop the min within the component as it breaks the intended behaviour to fix one use case.

For example, in the B version of the PDP experiment, the calendar is also used in the non-modal contact form (mad it one unit smaller to show the effect better) by dropping the min-width the calendar gets squeezed to the row and does not show up as in intended anymore.

Without min-width
![Screenshot 2021-05-06 at 09 51 12](https://user-images.githubusercontent.com/3371760/117262830-f4f1bf80-ae51-11eb-9ff8-b9a4b971bfc4.png)

Current implmentation
![Screenshot 2021-05-06 at 09 52 19](https://user-images.githubusercontent.com/3371760/117262822-f4592900-ae51-11eb-8bf3-73953526236b.png)

I am not entirely sure how I would approach it ( also depends on the desired behaviour of the designer) but I think we can not work around the bug by dropping the min width

Reply:
I just checked with the designer, we decided to keep the min-width and center the calendar if possible

Reply:
yeah I didnt know why min-width was needed thats why asked about it